src/pages/ItemDetail/ItemDetail.js

```diff
@@ -1,8 +1,157 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import './ItemDetail.scss';
 
 const ItemDetail = () => {
-  return <div />;
+  const [data, setData] = useState([]);
+  useEffect(() => {
+    setData(ITEM_DATA);
+  }, []);
+
+  const [quantity, setQuantity] = useState(1);
+  const price = data.price;
+
+  const plusPrice = () => {
+    setQuantity(quantity + 1);
+  };
+  const minusPrice = () => {
+    setQuantity(quantity - 1);
+  };
+  const disabledMinus = quantity < 2;
+
+  const [currentTab, setCurrentTab] = useState('상품상세');
+
+  const updateCurrentTab = ({ target }) => {
+    const currentTabName = target.innerHTML;
+    setCurrentTab(currentTabName);
+  };
+  return (
+    data.length === undefined && (
+      <>
```

Comment:
현재 data의 초기값이 배열이고, 실제 데이터는 객체이기 때문에 위 리뷰처럼 초기값도 객체로 수정하셔야 합니다.
그럴 경우에, 객체가 빈 객체인 경우에는 로딩 컴포넌트라던가 메시지를 표기해주는 방식으로 구현하시면 좋을 것 같네요!
예시를 들면 다음처럼 구현할 수 있습니다.
```suggestion
  if(Object.keys(data).length === 0) return <>loading...</>;
  
  return (
    <>
```